wallet/main.go

```diff
@@ -455,6 +456,37 @@ func main() {
 			fmt.Printf("%d key(s) verified and %d key(s) potentially corrupt\n",
 				verified, corrupt)
 
+		case "export":
+			pubKeys, err := wallet.GetKeys()
+			if err != nil {
+				fmt.Printf("Error: %s\n", err)
+				break
+			}
+			if len(pubKeys) == 0 {
+				fmt.Printf("No private keys found\n")
+				break
+			}
+			name := "export.txt"
+			f, err := os.Create(name)
+			if err != nil {
+				fmt.Printf("Error: %s\n", err)
+				break
+			}
+			count := 0
+			for _, pubKey := range pubKeys {
+				private, err := wallet.GetPrivateKey(pubKey)
+				if err != nil {
+					break
+				}
+				pair := fmt.Sprintf("%s\n%s\n\n",
```

Comment:
I'd maybe separate these with a comma and then a single new line. It'll be easier to use the file with standard tools that scan for single new lines like [`bufio.Scanner`](https://golang.org/pkg/bufio/#Scanner)

Reply:
Changed to "%s,%s\n".

I wonder if it makes sense to define a structure for this and then output as JSON; 
```
type Wallet struct {
    Address ed25519.PublicKey
    Private ed25519.PrivateKey
}
```
It's slightly less human-readable, but perhaps easier to ingest across different platforms.

Reply:
I think the way it's written now is easiest. Don't even need a JSON parser!